public/stylesheets/style.css

```diff
@@ -220,11 +220,69 @@ footer {
   background-color: #356635;
 }
 
-/*Styles for services ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+/*Styles for services ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
 #serviceno {
     margin-left: 1em;
 }
 
 #serviceyes {
     margin-right: 1em;
 }
+
+/*Styles for toggle switch ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
```

Comment:
You could consider making a css file just for sliders since it seems relatively long.